app/src/Page.php

```diff
@@ -2,7 +2,13 @@
 
 namespace {
 
+    use DNADesign\Elemental\Extensions\ElementalPageExtension;
```

Comment:
doesn't seem like we need all those imports here?

Reply:
Done